### PR TITLE
[minor] Explain the behavior of server.close() more precisely (#1341)

### PR DESCRIPTION
**Pull request by anliting**

### CODE REVIEW

File: doc/ws.md [hunk 1]
```diff
@@ -139,7 +139,11 @@ returned as a string.
 
 ### server.close([callback])
 
-Close the server and terminate all clients, calls callback when done.
+Close the server and terminate all clients, calls callback when done. The
```
Comment (lpinca) [inline doc/ws.md]: Can you please reword like this?

```
Close the HTTP server if created internally, terminate all clients and call
callback when done. If an external HTTP server is used via the `server` or
`noServer` constructor options, it must be closed manually.
```

I think it's more accurate.

Thank you.

Comment (anliting) [reply doc/ws.md]: Okay, I have updated it.

**Comment by lpinca** [comment]
Thank you.

**Comment by anliting** [comment]
Thank you too. `ws` is a great work.

**Pull request merged**